### PR TITLE
Fix types for validation method.

### PR DESCRIPTION
**Pull request by alexmacarthur**
# Description of Proposed Changes

# Related Issue (if applicable)

# Testing Steps

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -243,6 +243,6 @@ function Steppp(element: HTMLElement, options: any = defaultOptions) {
   };
 }
 
-Steppp.stepIsValid = (_slide: HTMLElement): boolean => true;
+Steppp.stepIsValid = async (_step: HTMLElement): Promise<boolean> => true;
```
Comment (JZDesign) [inline src/index.ts:246]: Should there be tests for this change?

When I see "Fix" and no tests… it makes me concerned.

Comment (alexmacarthur) [reply src/index.ts:246]: Ya know what... I dug into this, found out this method is not being used _anywhere_, and it's not mentioned in the public API anywhere. I'm gonna remove it and touch up some tests.

File: README.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ These are the events available to you:
 
 There's a demo out on StackBlitz. Tinker as much as you want!
 
-| Technology | Demo |
-| --- | --- |
-| Vanilla JS | [stackblitz.com/edit/steppp](https://stackblitz.com/edit/steppp) |
-| React | [stackblitz.com/edit/steppp-react-m6oyff](https://stackblitz.com/edit/steppp-react-m6oyff) |
+| Technology | Demo                                                                                       |
```
Comment (alexmacarthur) [inline README.md:240]: Prettier got a hold of this.

File: src/index.ts [hunk 3]
```diff
@@ -243,6 +243,4 @@ function Steppp(element: HTMLElement, options: any = defaultOptions) {
   };
 }
 
-Steppp.stepIsValid = (_slide: HTMLElement): boolean => true;
-
```
Comment (alexmacarthur) [inline src/index.ts:247]: Not actually used anywhere... I think this was carried over from the cowboy version we had built in the past.

**Review by JZDesign — APPROVED** [review]

**Pull request merged**